tests/testChain.cpp

```diff
@@ -0,0 +1,84 @@
+/* ----------------------------------------------------------------------------
+ * GTDynamics Copyright 2021, Georgia Tech Research Corporation,
+ * Atlanta, Georgia 30332-0415
+ * All Rights Reserved
+ * See LICENSE for the license information
+ * -------------------------------------------------------------------------- */
+
+/**
+ * @file  testChain.cpp
+ * @brief Test Chain class.
+ * @author: Dan Barladeanu, Frank Dellaert
+ */
+
+#include <CppUnitLite/TestHarness.h>
+#include <gtsam/base/Matrix.h>
+#include <gtsam/geometry/Pose3.h>
+
+#include "gtdynamics/dynamics/Chain.h"
+
+using namespace gtdynamics;
+using gtsam::assert_equal;
+using gtsam::Matrix;
+using gtsam::Point3;
+using gtsam::Pose3;
+using gtsam::Rot3;
+
+// Test Chain class functionality with a three-joint chain
```

Comment:
Generally when we define tests, the test function name is `TEST(TestModule, SpecificTest)`. So in this case, you should have `TEST(Chain, ThreeLinks)`.

Be sure to capitalize the specific test name, because on compilation, the full test is printed as `ChainThreeLinks` which is easier to read than `ChainthreeLinks`.

Reply:
done